### PR TITLE
[GIE Doc] Refine Docs for Cypher

### LINKED ISSUE #2987: The Cypher-related contents in GIE docs
We have currently supported Cypher in GIE. However, the Cypher-related documentations in GIE are still in an inconsistent/incomplete form. For example, users may not be aware of the support of Cypher. Therefore, it is important that we sort out the Cypher-related contents in GIE docs. Specifically, 
- [ ] Add Cypher contents in overview: https://graphscope.io/docs/latest/overview/graph_interactive_workloads
- [ ] Add Cypher query instruction in getting started: https://graphscope.io/docs/latest/overview/graph_interactive_workloads
- [ ] Refine how to expose Cypher's endpoint (bolt) in GIE deployments, such as in https://graphscope.io/docs/latest/interactive_engine/dev_and_test, and https://graphscope.io/docs/latest/interactive_engine/deployment

### PR DESCRIPTION
**Pull request by shirly121**
## What do these changes do?
as titled.

## Related issue number

Fixes #2987

### CODE REVIEW

File: docs/interactive_engine/deployment.md [hunk 1]
```diff
@@ -74,10 +74,11 @@ deployment and management of applications. To deploy GIE standalone using Helm,
    kubectl describe svc [YOUR_RELEASE_NAME]-gie-standalone-frontend \
    | grep "Endpoints:" | awk -F' ' '{print $2}'
    ```
-   You should see the GIE Frontend service endpoint as `<ip>:<gremlinPort>`.
+   You should see two exposed endpoints for GIE Frontend service, one is `<ip>:<gremlinPort>` for gremlin querying, another is `<ip>:<cypherPort>` for cypher querying.
```
Comment (longbinlai) [inline docs/interactive_engine/deployment.md:77]: Explicitly use two commands, one for obtaining Cypher endpoint, the other for Gremlin endpoint.

Comment (shirly121) [reply docs/interactive_engine/deployment.md]: done

File: docs/interactive_engine/dev_and_test.md [hunk 2]
```diff
@@ -131,9 +134,9 @@ java -cp ".:$GIE_TEST_HOME/lib/*" -Djna.library.path=$GIE_TEST_HOME/lib com.alib
 ```
 
 With the frontend service, you can open the gremlin console and set the endpoint to
-`localhost:8182`, as given [here](./tinkerpop_gremlin.md#gremlin-console).
+`localhost:8182`, as given [here](./tinkerpop/tinkerpop_gremlin.md#connecting-via-gremlin-console). Similarly, you can open the cypher-shell and set the url to `neo4j://localhost:7687` by using `-a` option, as given [here](./neo4j/cypher_sdk.md#connecting-via-cypher-shell).
 
-7. Kill the services of `vineyardd`, `gaia_executor` and `frontend`:
+1. Kill the services of `vineyardd`, `gaia_executor` and `frontend`:
```
Comment (longbinlai) [inline docs/interactive_engine/dev_and_test.md:139]: Why set this to no. 1

File: docs/interactive_engine/neo4j/cypher_sdk.md [hunk 3]
```diff
@@ -3,7 +3,8 @@ This document will provide you with step-by-step guidance on how to connect your
 FrontEnd service, which offers functionalities similar to the official Tinkerpop service.
 
 Your first step is to obtain the Bolt Connector of GIE Frontend service:
```
Comment (longbinlai) [inline docs/interactive_engine/neo4j/cypher_sdk.md:5]: 这里改成 Your first step is to obtain the Cypher endpoint (the [Bolt](https://neo4j.com/docs/bolt/current/bolt/) connector)

File: docs/interactive_engine/getting_started.md [hunk 4]
```diff
@@ -102,7 +102,7 @@ You could pass additional key-value pairs to customize the startup configuration
 
 ```python
 # Set the timeout value to 10 min
-g = gs.gremlin(graph, params={'pegasus.timeout': 600000})
+g = gs.gremlin(graph, params={'query.execution.timeout.ms': 600000})
```
Comment (longbinlai) [inline docs/interactive_engine/getting_started.md:105]: How this may be configured with the new python apis?

**Review by longbinlai — APPROVED** [review]

**Pull request merged**